.gitignore

```diff
@@ -37,3 +37,6 @@ node_modules
 
 # Ansible files
 /provisioning/playbook.retry
+
+# package lock file
+package-lock.json
```

Comment:
It would be my preference to commit `package-lock.json` to version control.

Reply:
Here is our thinking (from the earlier Nexus org) on why we considered this was undesirable - https://issues.gpii.net/browse/GPII-3098 - I think I still support this reasoning